src/graph/expression_operators.h

```diff
@@ -3,75 +3,346 @@
 #include "graph/node_initializers.h"
 
 namespace marian {
-
+/**
+ * @defgroup graph_ops Expression Graph Operators
+ *
+ * @{
+ */
```

Comment:
Is it possible to make it shorter? What do you think about:

    /** @defgroup ... */
    /** @{ */

or even a one-liner? If it is supported.

Reply:
As far I as recall, the new line was necessary. Certainly I had issues adding name groups to already existing groups when I tried more compact syntax. 

However, in the long run, I think this grouping may be more appropriate in the sphinx of #803.

Reply:
OK, thanks. I agree that the grouping may be useful for sphinx-generated docs.